interchain-exchange/tutorial/03-orderbook.md

```diff
@@ -154,20 +154,21 @@ Modify the `sellOrderBook.proto` file to have the fields for creating a sell ord
 The proto definition for the `SellOrderBook` should look like follows:
 
 ```proto
-// proto/sellOrderBook.proto
+// proto/ibcdex/sellOrderBook.proto
 syntax = "proto3";
-package tendermint.interchange.ibcdex;
+package username.interchange.ibcdex;
 
-option go_package = "github.com/tendermint/interchange/x/ibcdex/types";
+option go_package = "github.com/username/interchange/x/ibcdex/types";
 
-import "ibcdex/order.proto";
+import "ibcdex/order.proto"; // <--
 
 message SellOrderBook {
+  string creator = 1;
```

Comment:
We don't need the creator actually for this structure. But it actually made me think that for the next version of Starport we must provide the `--no-message` option to not generate this field (because the order book is created through internal on-chain logic and not directly by a message sent from a user)

Reply:
I see, the reason I added it is because the code generated with `starport type sellOrderBook orderIDTrack:int amountDenom priceDenom --indexed --module ibcdex`  [at this step](https://tutorials.cosmos.network/interchain-exchange/tutorial/02-app-init.html#create-the-transaction-types) contains reference to the creator fields and later on in the tutorial  I had compilation errors  if I removed the field

Reply:
Mmh you may have used Starport on `develop` branch maybe

Reply:
I actually realize that even with `--no-message` option `Creator` field is scaffolded, we can actually keep this line 👍